scripts/ci-e2e.sh

```diff
@@ -71,6 +71,38 @@ export ARTIFACTS="${ARTIFACTS:-${REPO_ROOT}/_artifacts}"
 export SKIP_RESOURCE_CLEANUP=false
 export USE_EXISTING_CLUSTER=false
 
+# Configure local output
+ARTIFACTS_LOCAL="${ARTIFACTS}/local"
```

Comment:
nit
s/local/host

Reply:
Done

Reply:
That's what it looked like, I wasn't sure that was intended. 

@fabriziopandini which one did you mean, localhost or host?
(I'll just change it, I don't really favor one over the other)

Reply:
`s/string1/string2` usually means search for `string1` and replace it with `string2`.

Reply:
Yeah I know, thx ;). `host` just didn't make sense to me :)

Reply:
@sbueringer sorry for being criptic. I was looking at 
<img width="269" alt="Screenshot 2021-03-31 at 17 45 26" src="https://user-images.githubusercontent.com/5013060/113172725-fd4e4d80-9248-11eb-9da8-dd23f2095288.png">
And in that context I found `local` a little bit confusing.
So the fist thing that came to my mind was "this is the host where the test runs", so I suggested `host`...
But this is really a nit, feel free to leave as it is (`localhost` is fine) or revert to `local`

Reply:
Yup local wasn't good. Then I'll keep localhost

Reply:
What does local, localhost, or host mean in this context?

Reply:
The idea is to get logs from the host which is running the `./scripts/ci-e2e.sh` shell script. 

In CI this is the Pod of the ProwJob, but of course I can also run this script on my laptop.

The idea was to name the folder in a way so that it makes clear that these are local logs.

Reply:
Can't really think of a better name, but one way to make it super clear is to add a log statement to a readme file that explains what this folder is, ie.:

```
echo "This folder contains logs from the host where the test ran" "${ARTIFACTS_LOCAL}/README" || true
```

(or something like it)

Reply:
Great idea, added.